bottlecap/src/lifecycle/invocation/processor.rs

```diff
@@ -23,7 +23,13 @@ use crate::{
     telemetry::events::{ReportMetrics, Status},
     traces::{
         context::SpanContext,
-        propagation::{DatadogCompositePropagator, Propagator},
+        propagation::{
+            text_map_propagator::{
+                DatadogHeaderPropagator, DATADOG_PARENT_ID_KEY, DATADOG_SPAN_ID_KEY,
+                DATADOG_TRACE_ID_KEY,
+            },
+            DatadogCompositePropagator, Propagator,
```

Comment:
At point in the invocation is this happening? Is this extracting context from the inbound event? Or is it getting it from the start invocation request headers?

Just asking cuz it doesn't look like we have support for w3c headers. Is that correct?

Reply:
I'm not sure which line you're talking about, you left the comment in the import

Reply:
> Just asking cuz it doesn't look like we have support for w3c headers. Is that correct?
We do

Reply:
Ahhh, didn't realize this is just the import. Thanks for answering the question!